tests/unit_tests/ensemble_evaluator/test_ensemble_evaluator.py

```diff
@@ -12,11 +18,58 @@
     FORWARD_MODEL_STATE_RUNNING,
 )
 
-from .ensemble_evaluator_utils import send_dispatch_event
+from .ensemble_evaluator_utils import TestEnsemble, send_dispatch_event
 
 
-def test_new_monitor_can_pick_up_where_we_left_off(evaluator):
-    evaluator._start_running()
+@pytest.fixture(name="evaluator_handle")
+def start_ee_in_thread(make_ee_config):
+    msg_queue = queue.Queue()
+    ee_async = None
+
+    def _ee_start():
+        async def start_ee_async():
+            nonlocal ee_async, msg_queue
+            ensemble = TestEnsemble(0, 2, 2, id_="0")
+            ee_async = EnsembleEvaluatorAsync(ensemble, make_ee_config(), 0)
+            msg_queue.put(ee_async)
+            run_task = asyncio.create_task(
+                ee_async.run_and_get_successful_realizations()
+            )
+            await ee_async._server_started.wait()
+            msg_queue.put("STARTED")
+            num_realizations = await run_task
+            msg_queue.put(num_realizations)
+
+        loop = new_event_loop()
+        asyncio.set_event_loop(loop)
+        loop.run_until_complete(start_ee_async())
+        loop.close()
+
+    loop_thread = ErtThread(target=_ee_start)
+    loop_thread.start()
+    yield msg_queue
+    ee_async.stop()
+    loop_thread.join()
+
+
+@pytest.fixture(params=["EnsembleEvaluatorAsync", "EnsembleEvaluator"])
+def evaluator_to_use(request):
+    ee_class = request.param
+
+    if ee_class == "EnsembleEvaluator":
+        evaluator = request.getfixturevalue("evaluator")
+        evaluator._start_running()
+    elif ee_class == "EnsembleEvaluatorAsync":
+        evaluator_handle = request.getfixturevalue("evaluator_handle")
+        evaluator = evaluator_handle.get()
```

Comment:
Is there a better way to pass EnsembleEvaluator without putting it as a message in the message queue?

Reply:
The message queue is thread safe and thus is ideal for sending / passing variables from async context to sync one.

Reply:
Okay, but can we add typing to line 63 then? Maybe something like 
`evaluator_handle: queue.Queue[Union[str, EnsembleEvaluator]]` would work?

Reply:
I have moved the fixture to conftest and there are not annotations, so will leave it as is I guess